test/unit/validators/array_list_test.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:test/test.dart';
+import 'package:utopia_framework/utopia_validators.dart';
+
+import 'mock_validator.dart';
+
+String _description(String value) {
+  return 'Value must be a valid array and $value';
+}
+
+void main() {
+  final mockValidator = MockValidator();
```

Comment:
I know it makes this dependent to other validators, but let's instead test this with real Text, and other validators we already have ?

Reply:
But won't that be against the ethics of unit testing? Passing real dependencies in test? Also, I thought people can learn how to create Mocks on their own. So, I on purpose created it manually without using Mocktail. What do you think?

Reply:
Okay thats all right, let's later add one in e2e test that tests ArrayList validator with real validators.